source/developers/how-tos/maintain-a-repo.rst

```diff
@@ -145,6 +145,77 @@ outlined in :doc:`OEP-54 <openedx-proposals:processes/oep-0054-core-contributors
 
 .. _CCs Wiki Page: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Core+Contributors+to+the+Open+edX+Project
 
+Hold Yourself to Community Standards
+************************************
+
+A prerequisite for a Maintainer is to be a Core Contributor.  Part of what this
+means is that you must hold yourself to the same standards as any CC - any
+contributor, really - in your repository.  You are still a member of the
+community, after all!
+
+Here are a few rules-of-thumb to keep in mind:
+
+* Participate!  Make yourself available in the several community communication
+  channels - Discourse, Slack, working group meetings, and, of course, Github
```

Comment:
Should we make Discourse & Slack links? eg https://discuss.openedx.org/ and https://openedx.org/slack